.github/workflows/dependabot-automation.yml

```diff
@@ -0,0 +1,27 @@
+name: Dependabot automation
+
+on: pull_request
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  merge-auspice-bump:
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - id: metadata
+        uses: dependabot/fetch-metadata@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      # Run if Auspice is an updated dependency and it is not a major version bump
+      # to allow for manual testing before merging.
+      - if: |
+          contains(fromJSON(steps.metadata.outputs.updated-dependencies-json).*.dependencyName, 'auspice')
+          && steps.metadata.outputs.update-type != 'version-update:semver-major'
```

Comment:
_non-blocking, informational only_

This top-level `update-type` will be the aggregate (sort of a `max()`) of everything updated, which may include deps other than Auspice, right? But I guess in practice, for now, only Auspice will appear because of our current dependabot config?

The `auspice` entry in `updated-dependencies-json` also has its own specific `update-type`, but it looks like it's not possible in GitHub's expression syntax to extract that entry from the array, so we can't address its `update-type` in a condition. (We could extract it elsewhere, in another job step, but that seems unnecessary for now I guess.)

Reply:
Oh I didn't realize that, thanks. I updated the comment in [force-push](https://github.com/nextstrain/nextstrain.org/compare/670b3b0df07d9792437e50fe67696475f4a2033b..2bb5028d6e2ae143354192a9771c93aacdc34093) to be more accurate.